eos_data_distribution/ndn/base.py

```diff
@@ -142,6 +142,7 @@ def _expressInterest(self, interest, forever=False,
         if not onTimeout:
             onTimeout = partial(self.onTimeout, forever=forever)
 
+        interest.setInterestLifetimeMilliseconds(20000)
```

Comment:
all these are demo commits, i'm going to delete them. sorry you took time to review them.